client/src/pages/RegisterPage.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import axios from "axios";
+
+const RegisterPage = () => {
+  const [formData, setFormData] = useState({
+    name: "",
+    email: "",
+    password: "",
+  });
+
+  // handle user input
+  const handleChange = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  // handle submit
+  async function registerUser(ev) {
+    ev.preventDefault();
+    try {
+      await axios.post("/register", {
+        name,
+        email,
+        password,
+      });
+      alert("Registration successful. Now you can log in");
```

Comment:
Is there a better way than using an alert for this?

Reply:
I'm still working on the RegisterPage. The alert is a temporary solution to test that the API call is functioning correctly. Moving forward, I plan to display a message element on the page, which will provide a more consistent and user-friendly experience.

Reply:
@ericadev I'm still working on the RegisterPage. The alert is a temporary solution to test that the API call is functioning correctly. Moving forward, I plan to display a message element on the page, which will provide a more consistent and user-friendly experience.